File: Hoteles-Reserva-WS/src/java/models/Module.java
package models;

public class Module {
    private int idModule, idFatherModule;
    private String name, description, path, level, order, dateCrea, dateMod;
    private Boolean active;

    public int getIdModule() {
        return idModule;
    }

    public void setIdModule(int idModule) {
        this.idModule = idModule;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public String getLevel() {
        return level;
    }

    public void setLevel(String level) {
        this.level = level;
    }

    public String getOrder() {
        return order;
    }

    public void setOrder(String order) {
        this.order = order;
    }

    public String getDateCrea() {
        return dateCrea;
    }

    public void setDateCrea(String dateCrea) {
        this.dateCrea = dateCrea;
    }

    public String getDateMod() {
        return dateMod;
    }

    public void setDateMod(String dateMod) {
        this.dateMod = dateMod;
    }

    public int getIdFatherModule() {
        return idFatherModule;
    }

    public void setIdFatherModule(int idFatherModule) {
        this.idFatherModule = idFatherModule;
    }

    

    public Boolean getActive() {
        return active;
    }

    public void setActive(Boolean active) {
        this.active = active;
    }
    
}

File: Hotel-Reservacion/Hotel-Reservacion/Models/Module.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Hotel_Reservacion.Models
{
    public class Module
    {
        private int idModule, idFatherModule;
        private String name, description, path, level, order, dateCrea, dateMod;
        private bool active;

        public int IdModule
        {
            get
            {
                return idModule;
            }
            set
            {
                idModule = value;
            }
        }
        public String Name
        {
            get
            {
                return name;
            }
            set
            {
                name = value;
            }
        }
        public String Description
        {
            get
            {
                return description;
            }
            set
            {
                description = value;
            }
        }
        public String Path
        {
            get
            {
                return path;
            }
            set
            {
                path = value;
            }
        }
        public String Level
        {
            get
            {
                return level;
            }
            set
            {
                level = value;
            }
        }
        public String Order
        {
            get
            {
                return order;
            }
            set
            {
                order = value;
            }
        }
        public int IdFatherModule
        {
            get
            {
                return idFatherModule;
            }
            set
            {
                idFatherModule = value;
            }
        }
        public String DateCrea
        {
            get
            {
                return dateCrea;
            }
            set
            {
                dateCrea = value;
            }
        }
        public String DateMod
        {
            get
            {
                return dateMod;
            }
            set
            {
                dateMod = value;
            }
        }
        public bool Active
        {
            get
            {
                return active;
            }
            set
            {
                active = value;
            }
        }
    }
}